ibis/backends/tests/test_dot_sql.py

```diff
@@ -45,17 +46,20 @@
     ],
 )
 def test_con_dot_sql(backend, con, schema):
-    alltypes = con.table("functional_alltypes")
+    alltypes = backend.functional_alltypes
     # pull out the quoted name
-    name = _NAMES.get(con.name, alltypes.op().name)
+    name = _NAMES.get(con.name, "functional_alltypes")
+    quoted = getattr(getattr(con, "compiler", None), "quoted", True)
+    dialect = _IBIS_TO_SQLGLOT_DIALECT.get(con.name, con.name)
+    cols = [
```

Comment:
Required due to backend specific quoting?

Reply:
Yes. This effectively makes identifiers case sensitive